src/foam/dao/pg/ConnectionPool.java

```diff
@@ -0,0 +1,32 @@
+package foam.dao.pg;
+
+import org.apache.commons.dbcp2.BasicDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class ConnectionPool {
```

Comment:
Make this a modelled object so it can be configured through the GUI. Then make it a service so it can be accessed from the Context and reused by DAO's.

Reply:
What exactly am I modelling here?

Reply:
Okay I've modelled ConnectionPool and made it a NanoService. I'm fetching form the context in PostgresDAO now too.